x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_actions/management.js

```diff
@@ -37,7 +47,7 @@ export function actionsMenuContent(
         defaultMessage: 'Start datafeed',
       }),
       icon: 'play',
-      enabled: (item) => item.deleting !== true && canStartStopDatafeed,
+      enabled: (item) => isJobBlocked(item) === false && canStartStopDatafeed,
```

Comment:
I don't see these action items disabled after hitting reset. Is this to be expected, without the planned follow-up for polling?

![reset_actions](https://user-images.githubusercontent.com/7405507/129066762-c5981934-5e5d-4fe8-a8e0-1d1fcc9adf04.gif)

Reply:
https://github.com/elastic/kibana/pull/108039/commits/797c2746a82dab24184acf901b36b6910d3e2bf6 fixes this issue.
It's still tricky to reproduce as you need to catch the job while the reset task is running.
I think i might need to reinstate the slight delay when refreshing the jobs after applying the reset, just to increase the chance that the first refresh will catch the the job why the task is running, and no before, which is happening now.
The follow up work will add a poll for the tasks so the jobs list will update once the task is complete.